core_backend/deployment/bootstrap_demo.sh

```diff
@@ -23,23 +23,31 @@ function get_secret_value() {
 
 echo "Fetching variables from aws store.."
 
-# This is the name of the secret in aws secrets manager
-SECRET_JWT="aaq-demo-jwt-secret"
-SECRET_CONTENT="aaq-demo-content-access"
-SECRET_WHATSAPP="aaq-demo-whatsapp-token"
-SECRET_WEB_DB_CONNECTION="aaq-demo-web-db-connection-details"
-
-export NEXT_PUBLIC_BACKEND_URL=https://aaq-demo.idinsight.io/api
+# This is the name of the secrets in aws secrets manager.
+# The secrets are created by the infrastructure module
+# The values should be the same as the ones in the infrastructure module
+SECRET_JWT="${PROJECT_NAME}-${ENV}-jwt-secret"
+SECRET_CONTENT="${PROJECT_NAME}-${ENV}-content-access"
+SECRET_WHATSAPP_VERIFY="${PROJECT_NAME}-${ENV}-whatsapp-verify-token"
+SECRET_WEB_DB_CONNECTION="${PROJECT_NAME}-${ENV}-web-db-connection-details"
+SECRET_WHATSAPP="${PROJECT_NAME}-${ENV}-whatsapp-token"
+SECRET_OPENAI="${PROJECT_NAME}-${ENV}-open-ai-key"
+SECRET_QUESTION_ANSWER="${PROJECT_NAME}-${ENV}-question-answer"
+
```

Comment:
Made changes to the secrets in bootstrap. Thought that this would be the default bootstrap then if we need to make specific changes to a project or environment, we can create a new bootstrap